octopoes/octopoes/api/router.py

```diff
@@ -412,10 +412,15 @@ def get_scan_profile_inheritance(
 ) -> list[InheritanceSection]:
     ooi = octopoes.get_ooi(reference, valid_time)
     if not ooi.scan_profile:
-        raise HTTPException(status_code=status.HTTP_400_BAD_REQUEST, detail="OOI does not have a scanprofile")
+        raise HTTPException(
```

Comment:
Are these format-only changes? Then these would probably be reverted when https://github.com/minvws/nl-kat-coordination/pull/2975 is merged?